packages/wdio-cli/src/templates/snippets/mocha.ejs

```diff
@@ -1,14 +1,11 @@
 <%  if (answers.framework === 'mocha') { %>
-    // Options to be passed to Mocha.
-    // See the full list at http://mochajs.org/
```

Comment:
Similar to above, lets have a link to the Mocha options, https://webdriver.io/docs/frameworks#mocha-options

Reply:
same for Cucumber and Jasmine